Decisions.TruBot/Data/TruBotConstants.cs

```diff
@@ -0,0 +1,16 @@
+namespace Decisions.TruBot.Data
+{
+    public class TruBotConstants
+    {
+        internal const string TRUBOT_IMAGES_PATH = "../wwwroot/Content/CustomModuleImages/Decisions.TruBot/|trubot.svg";
+        
+        internal const string TRUBOT_ROOT_FOLDER_ID = "01HNB8V81CT9RSWA6ZB6S9ZRT0";
```

Comment:
Does this have to be specific to projects?

Reply:
The TruBot folder was requested to be in the Jobs and Events directory in the requirements doc

_1. Once the module is installed, this folder will live under the System folder and only be available for System Administrators
2. Location will be System > Jobs and Events_

Reply:
Please double check this with Jinesh. 
IMO it makes sense to make these project specific.
Projects Hub in general may not have been taken into account while designing this

Reply:
@jineshu What are your thoughts on this?